src/main/java/me/botsko/prism/utils/EnchantmentUtils.java

```diff
@@ -130,23 +136,30 @@ public static String getClientSideEnchantmentName(Keyed ench, int level) {
             enchName = "projectile protection";
         } else if (ench.equals(Enchantment.WATER_WORKER)) {
             enchName = "aqua affinity";
-        } else {
+        } else if (ench.equals(Enchantment.VANISHING_CURSE)) {
+            enchName = "vanishing curse";
+        }else {
```

Comment:
Codacy found an issue: [WhitespaceAround: '}' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5766072)